examples/cross.js

```diff
@@ -0,0 +1,69 @@
+var data = require('./data/data.json')
+var streamSift = require('../lib')
+var c = require('colors/safe')
+
+/**
+* $cross
+* $crossOrEqual
+*/
+
+var spec = { input : { $c : 1 } }
+var ss = streamSift(spec)
+
+data.map(function(obj){
+  stdout("$cross", spec, obj, ss(obj))
+})
+
+var spec = { input : { $ce : 1 }}
+var ss = streamSift(spec)
+
+data.map(function(obj){
+  stdout("$crossOrEqual", spec, obj, ss(obj))
+})
+
+/**
+* $crossGreaterThan
+* $crossGreaterThanOrEqual
+* $crossLessThan
+* $crossLessThanOrEqual
+*/
+
+var spec = { input : { $cgt : 1 }}
+var ss = streamSift(spec)
+
+data.map(function(obj){
+  stdout("$crossGreaterThan",spec, obj, ss(obj))
+})
+
+var spec = { input : { $cgte : 1 }}
+var ss = streamSift(spec)
+
+data.map(function(obj){
+  stdout("$crossGreaterThanOrEqual", spec, obj, ss(obj))
+})
+
+var spec = { input : { $clt : 2 }}
+var ss = streamSift(spec)
+
+data.map(function(obj){
+  stdout("$crossLessThan", spec, obj, ss(obj))
+})
+
+var spec = { input : { $clte : 2 }}
+var ss = streamSift(spec)
+
+data.map(function(obj){
+  stdout("$crossLessThanOrEqual", spec, obj, ss(obj))
+})
+
+
+//ugly console helper
+function stdout (fn, spec, obj, predicate){
+  var falsey = c.red
+  var truthy = c.green
+  console.log("function:    ", fn)
```

Comment:
I'm wondering if explicitly listing the function is helpful or confusing. From a new users perspective they shouldn't be thinking in terms of single functions but just general patterns. Many patterns will use more than one function. I find this confusing. Your next line of logging the `pattern` in whole seems much more intuitive. When you isolate a single function name it becomes very abstract and is no longer telling the story about pattern matching. I can see your goal here is to emphasize a runnable demo of different functions. Maybe then change your title format or just let the pattern speak for itself. People need to learn to feel comfortable reading a pattern and since this pattern is so short it can double as a title, effectively.